src/components/tabs.tsx

```diff
@@ -1,14 +1,58 @@
-import React from "react";
+import styled from "@emotion/styled";
+import React, { ButtonHTMLAttributes, FunctionComponent } from "react";
+
+const InnerButtonIcon = styled.button`
+  width: 40px;
+  height: 40px;
+  svg {
+    max-width: 20px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  @media print {
+    display: inline-block;
+    background-color: #4e73b6; // primary
+    box-shadow: 0 4px 6px 3px rgba(54, 81, 130, 0.4), 0 2px 4px 2px rgba(54, 81, 130, 0.2);
+    color: white;
+  }
+`;
+
+export const ButtonIcon: FunctionComponent<ButtonHTMLAttributes<any>> = ({
+  className,
+  children,
+  disabled,
+  ...props
+}) => {
+  return (
+    <InnerButtonIcon
+      className={`transition-colors duration-200 ease-out cursor-pointer font-bold p-2 rounded shadow-md border border-transparent text-primary hover:text-primary-dark bg-white hover:bg-gray-100 hidden md:inline-block ${className} ${
+        disabled && "cursor-not-allowed"
```

Comment:
any reason to hide in mobile?

Reply:
any reason to print on mobile ? :thinking: I don't know I felt it's unecessary

Reply:
looks like there's this airprint thing https://support.apple.com/en-us/HT201387

nevertheless this is edge case, ok with disabling it unless there is a need

trying on opencerts one and how it looks like:

![airprint](https://user-images.githubusercontent.com/4774314/114661813-30eda500-9d2a-11eb-8820-d120e6ef1f48.jpeg)